contracts/8.17/managers/PositionManager.sol

```diff
@@ -107,14 +107,16 @@ contract PositionManager is PausableUpgradeable, IManager {
 
   /// @dev Initializer for intializing PositionManager
   /// @param _bookKeeper The address of the Book Keeper
-  function initialize(address _bookKeeper, address _showStopper) external initializer {
+  function initialize(address _bookKeeper, address _showStopper, address _priceOracle) external initializer {
```

Comment:
Motivation is that address _priceOracle is used in many functions anyway. It makes more sense to configure it from the beginning.